localization/gyro_odometer/README.md

```diff
@@ -28,4 +27,13 @@
 
 ## Assumptions / Known limits
 
-TBD.
+- [Assumption] The frame_id of input twist message must be base_link
+
+- [Assumption] The covariances in the input messages must be properly entered
```

Comment:
I think it would be better to say like 

```
The covariance in the input messages must be properly assigned
```

or 

```
The covariance in the input messages must be properly initialized
```

Reply:
Thank you!
fixed https://github.com/autowarefoundation/autoware.universe/pull/412/commits/046f6aedb135a993d50bcaa19ca4c8042759d0ca